src/include/migraphx/op/split.hpp

```diff
@@ -0,0 +1,213 @@
+#ifndef MIGRAPHX_GUARD_OPERATORS_SPLIT_HPP
+#define MIGRAPHX_GUARD_OPERATORS_SPLIT_HPP
+
+#include <array>
+#include <migraphx/operation.hpp>
+#include <migraphx/check_shapes.hpp>
+#include <migraphx/stringutils.hpp>
+#include <migraphx/streamutils.hpp>
+#include <migraphx/literal.hpp>
+#include <migraphx/shape_for_each.hpp>
+#include <migraphx/config.hpp>
+#include <cmath>
+#include <utility>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace op {
+
+struct split
```

Comment:
Lets rename this to `horizontal_fusion_split` since its only for horizontal fusion.

Reply:
fixed